pkg/cloudprovider/vm.go

```diff
@@ -102,7 +97,6 @@ func (c *VMClient) Get(rgName string, nodeName string) (compute.VirtualMachine,
 
 	vm, err := c.client.Get(ctx, rgName, nodeName, "")
 	if err != nil {
```

Comment:
Please confirm that the error has indications whether it happened from Get or WaitForCompletionRef .